### ISSUE TITLE
Melhoria de mensagem: Parâmetro $numIdTipoProcedimento não informado.

### ISSUE THREAD

**Issue by yurioliveiramgi** [pengovbr/mod-sei-pen#760]
**Sua solicitação de melhoria está relacionada a um problema? Por favor descreva.**
Quando um processo é tramitado e o órgão de destino não possui um mapeamento ativo para o Tipo de Processo, o Tipo de Processo não existe no SPE de destino e nem o Tipo de Processo Padrão está configurado, uma recusa é apresentada para o trâmite com a seguinte mensagem:

_Parâmetro $numIdTipoProcedimento não informado._
![image](https://github.com/user-attachments/assets/40abda8b-d31c-4be6-8ac6-584c6f61efce)

-----

**Descreva a solução que você gostaria**
A mensagem deve carregar mais informações a respeito do motivo de recusa. Dessa forma, é solicitado que a mensagem seja alterada para:

"O Tipo de Processo [Nome do Tipo de Processo] não existe no sistema de destino. OBS: A recusa é uma das três formas de conclusão de trâmite. Portanto, não é um erro."

-----

**Definição de Feito (D.O.D. - Definition of Done)**

- [ ] Evidências do trabalho realizado.

-----

**Comment by yurioliveiramgi** [comment]
@euviniciusmendonca PSC

**Comment by amelquia** [comment]
14/01/25
Issue removida do backlog da sprint em virtude da priorização da compatibilização do SEI versão 5.0.
A issue retornou ao backlog geral.

**Comment by mrglaydson** [comment]
A mensagem sublinhada em vermelho exibe a antiga mensagem de recusa exibida quando não era encontrado o tipo de procedimento. Já a mensagem destacada em verde demonstra a nova versão após a modificação realizada.

![Image](https://github.com/user-attachments/assets/cbf28f5c-27f8-4483-92be-16838116a3f3)

**Comment by yurioliveiramgi** [comment]
@mrglaydson , uma dúvida:
"Módulo do Tramita:" é para identificar a origem da mensagem?

**Comment by mrglaydson** [comment]
@yurioliveiramgi Exatamente. Há duas semanas foi adicionada no branch master uma atualização que inclui o "Módulo do Tramita:" em todas as mensagens de erro/exeções do módulo. [ (commit ) ](https://github.com/pengovbr/mod-sei-pen/commit/6c44c39ae074e5189e0095686003198d1c60389f#diff-6d7c4d39e5a1fe4056a3f1096db24eeb134b6cb21f6bfaad41ecb33f0d4544e8L30)Isso foi feito para facilitar a identificação, permitindo diferenciar se a mensagem de erro é proveniente do core do SEI ou do módulo. Como essa atualização foi realizada após a criação desta issue, a imagem que você postou não traz. Nessa atualização, eu mantive e alterei a mensagem citada.

**Comment by yurioliveiramgi** [comment]
Perfeito @mrglaydson , obrigado pelo retorno.
Particularmente, de um ponto de vista estético eu colocaria "[Módulo do Tramita] - ", para não ficar ":" seguidos de ":" algumas vezes.

@euviniciusmendonca , para sua avaliação, a solução apresentada https://github.com/pengovbr/mod-sei-pen/issues/760#issuecomment-2607966795

**Comment by amelquia** [comment]
27/01/25
Issue transferida para a sprint atual 56 para que a Análise do P.O. seja realizada.

**Comment by amelquia** [comment]
21/02/25
Issue transferida para a sprint atual 57 para que a Análise do P.O. seja realizada.

**Comment by amelquia** [comment]
10/03/25
Issue transferida para a **Em Análise PO** para a Sprint 58.

**Comment by yurioliveiramgi** [comment]
Issue validada. As alterações estão de acordo com o esperado.
